Makefile

```diff
@@ -25,13 +25,19 @@ deps:
 		false; \
 	fi
 	curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+	curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+	sudo add-apt-repository \
+		"deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+		`lsb_release -cs` \
+		stable"
 	-sudo apt-get -qq update
 	if [ `lsb_release -rs` = "14.04" ]; then \
```

Comment:
I kept it because the installed version of python wasn't the same, it is python3.4-venv for trusty and python3-venv for other versions